setup.php

```diff
@@ -1229,6 +1219,17 @@ function cloneClicked() {
                                 echo "$ok<br />\n";
                                 flush();
                             }
+
+
+                            echo "Adding Initial User...\n";
+                            flush();
+                            if (! $installer->add_initial_user()) {
+                                echo "$error.\n";
+                                echo $installer->error_message;
+                                break;
+                            }
+                            echo "$ok<br />\n";
+                            flush();
                         }
```

Comment:
Isn't this missing the `install_additional_users()`?

Reply:
Yes it is, I've added it in my next push, but hopefully you can test this for me, I can't figure out the credentials to use for the setup.php

Reply:
This should now be fixed, let me know if it works for you.

Reply:
TODO for me: test the manual install

Reply:
tested setup.php and it works

![tenor99](https://user-images.githubusercontent.com/278968/107423733-8f4f9a80-6ad1-11eb-88e4-e1f3d6da2531.gif)